systems/plants/URDFRigidBodyManipulator.cpp

```diff
@@ -345,6 +339,29 @@ bool URDFRigidBodyManipulator::addURDF(boost::shared_ptr<urdf::ModelInterface> _
       dofnum[index-1] = _dofnum;
     }
 
+    if(l->second->inertial != nullptr)
+    {
+      boost::shared_ptr<urdf::Inertial> inertial = l->second->inertial;
+      bodies[index]->mass = inertial->mass;
+      bodies[index]->com << inertial->origin.position.x, inertial->origin.position.y, inertial->origin.position.z, 1.0;
+      Matrix3d I3d;
+      I3d << inertial->ixx, inertial->ixy, inertial->ixz,
+             inertial->ixy, inertial->iyy, inertial->iyz,
+             inertial->ixz, inertial->iyz, inertial->izz;
+
+      // from mcI.m
+      Vector3d com = bodies[index]->com.head(3);
+      Matrix3d C = vectorToSkewSymmetric(com);
+      bodies[index]->I.block(0,0,3,3) << I3d + bodies[index]->mass*C*C.transpose();
+      bodies[index]->I.block(0,3,3,3) << bodies[index]->mass*C;
+      bodies[index]->I.block(3,0,3,3) << bodies[index]->mass*C.transpose();
+      bodies[index]->I.block(3,3,3,3) << bodies[index]->mass*Matrix3d::Identity();
+
+      Transform<double, 3, 1> T;
+      poseToTransform(inertial->origin,T.matrix());
+      transformSpatialInertia(T,static_cast<  Gradient<Isometry3d::MatrixType, Eigen::Dynamic>::type* >(NULL),bodies[index]->I);
```

Comment:
From https://stackoverflow.com/questions/20509734/null-vs-nullptr-why-was-is-replaced:
nullptr is always of pointer type. If you have overloaded functions function foo(int) and foo(bar*), passing in NULL is ambiguous, nullptr disambiguates.